src/test/java/lembas/core/api/common/SimpleLoggerTest.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (C) 2020 Lembas Modding Team
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package lembas.core.api.common;
+
+import org.junit.Test;
+
+public class SimpleLoggerTest

```

Comment:
This should probably also contain tests for when nothing is wrong, and a test when you try to create a logger with a null name while a provider is set.

Reply:
Yeah, if we want to do the full testing, we've to account for a lot of test-cases, which indeed are exploding with more parameters. I just wanted to ensure the most important things, therefore the tests. I'll add the two conditions because they make sence there, but generally, I don't think that we should spend too much effort on the tests, yes, usually that's a good thing to put effort into, but I don't want to make the whole thing test-driven, but only use the tests for a subset of the use-cases.

Reply:
Fixed with b4c118bc5adbfca296b037f41550b3c982f90834